tests/plugins/piccolo_orm/piccolo_app.py

```diff
@@ -15,7 +15,7 @@
     Venue,
 )
 
-CURRENT_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+CURRENT_DIRECTORY = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Maybe we should use here pathlib for consistency?

Reply:
If we want to use pathlib only then `flake8-use-pathlib` seems to be a good addition to pre-commit hooks

Maybe, `flake8-use-fstring` is also worth checking out.